### DISCUSSION TITLE
Is there a javascript api to get the full config (or maybe just the classes)?

### DISCUSSION THREAD

**Discussion by NullVoxPopuli** [tailwindlabs/tailwindcss#5576] [Help] [answered]
I know there are 3 layers, base, components, and utilities. Is there a way to retrieve a list of these in javascript?

**Comment by dance2die** [comment, marked as answer]
To retrieve class names, check out https://github.com/tailwindlabs/tailwindcss/discussions/5086
For full configs, https://tailwindcss.com/docs/configuration#referencing-in-java-script

**Reply by NullVoxPopuli** [reply]
oh noice, thank you!
